node-src/git/getCommitAndBranch.test.ts

```diff
@@ -62,6 +71,10 @@ describe('getCommitAndBranch', () => {
   });
 
   it('retrieves CI context', async () => {
+    process.env.GITHUB_EVENT_NAME = 'push';
```

Comment:
All these tests were accidentally working since the event name was `push` in the past. This forces the behavior so they should work no matter how we trigger these workflows.